README.slack.md

```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
+1. Add the App to Slack
```

Comment:
Code level comment

Reply:
@mpsolano  Do you get called out in Slack?

Reply:
Or does it have to be @mikesolano ?